Source/scrc.f90

```diff
@@ -387,7 +387,7 @@ MODULE SCRC
                       NSCARC_STACK_MAX               = 10            !> maximum number of consecutive solvers
 
 REAL(EB), PARAMETER:: NSCARC_THRESHOLD_CONVERGENCE   = 1.0E-15, &    !> threshold for convergence
```

Comment:
@SusanKilian All your (EB) parameters should be eight byte reals, for example, 1.0E-15_EB

Reply:
Also, what is your intention for 1.0E-16?  Is this machine precision?  If so, we use use TWO_EPSILON_EB, which is a global parameter.